scripts/packaging/generate_asset_cache.py

```diff
@@ -73,8 +72,10 @@ def generate_asset_cache(tag):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 2:
-        sys.exit('Missing argument: commit sha or tag.')
+    if len(sys.argv) != 3:
+        sys.exit('''Missing one or more arguments.
+            It should have the form: generate_asset_cache [commit sha or tag] [webots version name]''')
     else:
         tag = sys.argv[1]
-    generate_asset_cache(tag)
+        version_name = sys.argv[2]
+    generate_asset_cache(tag, version_name)
```

Comment:
Why the webots version cannot be read directly from the file as it was done before?

Reply:
It could, as could be the tag, but I think that we never call this script directly in the build process (we call directly the generate_asset_cache function)

So as this function is for debugging purpose and that the first parameter is chosen by the user I thought that he could give the second as well.